packages/blade/src/components/Box/BaseBox/BaseBox.native.tsx

```diff
@@ -13,7 +17,28 @@ const isSupportedOnReactNativeElement = (prop: string): boolean => {
   return !prop.startsWith('padding') && !prop.startsWith('margin') && prop !== 'flex';
 };
 
-const BaseBox = styled(View)
+/*
```

Comment:
I wonder if we should simplify this whole implementation. Its getting a bit complex now. 

```tsx
const StyledBaseBox = styled(View)`...`

const BaseBox = (props: BaseBoxProps) => {
// all the checks like `isSupportedOnReactNativeElement`, etc.
const elevationStyles = {...}

return <StyledBaseBox {...props} {...elevationStyles}/>;
}
```

Reply:
umm yeah.. I don't know kya phoonk ke baitha tha, my brain thought I'll introduce 2 wrapping Boxes if i do it this way.